src/ms-store-badge.ts

```diff
@@ -80,6 +98,14 @@ class MSStoreBadge extends HTMLElement {
       this.updateImageSrc();
     } else if (name === "productid" && newValue !== oldValue && typeof newValue === "string") {
       this.productId = newValue;
+    } else if (name === "cid" && newValue !== oldValue && typeof newValue === "string") {
+      this.cid = newValue;
+    } else if (name === "windowmode" && (newValue === "popup" || newValue === "full") && oldValue !== newValue) {
+      this.windowMode = newValue;
+      this.updateImageSrc();
+    } else if (name === "theme" && (newValue == "dark" || newValue === "light" || newValue === "auto") && oldValue !== newValue) {
+      this.theme = newValue;
```

Comment:
Suppose the user sets `theme` to dark. Then, later, he removes the `theme` attribute altogether.

In this case, theme will still be set to `dark`. 

What should happen is, a null or undefined `theme` should be reset it to its default (auto).